docs/UserGuide.md

```diff
@@ -255,8 +255,8 @@ Format: `find [KEYWORD_1] [KEYWORD_2] ...`
 
 Examples:
 * `find CS2101` returns `CS2101 Oral Presentation 1`
-* `find CS1231S Mission` returns `CS1101S Mission`, `CS1231S Graded Assignment`<br>
-  ![result for 'find CS1231S Mission'](images/findCS1231SmissionResult.png)
+* `find CS2103T Quiz` returns `CS2103T Project`, `CS1101S Quiz`<br>
+  ![result for 'find CS1231S Mission'](images/findCS2103TQuizResult.png)
```

Comment:
"![result for 'find CS1231S Mission']" would need to be changes to the CS2103 version too.